pkg/chezmoi/chezmoi_unix.go

```diff
@@ -26,6 +26,9 @@ func FQDNHostname(fileSystem vfs.FS) string {
 	if fqdnHostname, err := etcHostsFQDNHostname(fileSystem); err == nil && fqdnHostname != "" {
 		return fqdnHostname
 	}
+	if fqdnHostname, err := etcMynameFQDNHostname(fileSystem); err == nil && fqdnHostname != "" {
```

Comment:
As only OpenBSD uses this, could you please put this inside an `if runtime.GOOS == "openbsd" { ... }` conditional.

Reply:
Done as well!